openstack_controller/datadog_checks/openstack_controller/config.py

```diff
@@ -79,7 +83,7 @@ def _validate_microversion(self, microversion, service):
             is_float = float(microversion)
         except Exception:
             pass
-        if not is_latest and not is_float:
+        if not is_latest and not is_float and 'volume' not in microversion:
```

Comment:
What is checking this new condition?

Reply:
The method of specifying microversion in Cinder is by adding a header of the following format: [OpenStack-API-Version: volume <version>](https://docs.openstack.org/cinder/latest/contributor/api_microversion_history.html)